test/unit/component.test.js

```diff
@@ -932,7 +932,7 @@ QUnit.test('should provide interval methods that automatically get cleared on co
   assert.ok(intervalsFired === 5, 'Interval was cleared when component was disposed');
 });
 
-QUnit.test('should provide *AnimationFrame methods that automatically get cleared on component disposal', function(assert) {
+QUnit.test('should provide a requestAnimationFrame method that is cleared on disposal', function(assert) {
```

Comment:
Since we have a new animation frame method this test was confusing named `*AnimationFrame`.

Reply:
Do we have a test that verifies `requestAnimationFrame` is cleared on dispose? Since this test seems to have been updated to use `requestNamedAimtionFrame`. Maybe this test wasn't supposed to be updated?

Reply:
I should have mentioned this but this test was duplicated for `requestAnimationFrame` https://github.com/videojs/video.js/blob/master/test/unit/component.test.js#L1013-L1069 so I changed the duplicate into a test for `requestNamedAnimationFrame`

Reply:
Ok, cool, the tests do exist but was kind of confusing to find them.